Remove flag that is not accepted by older gcc versions.
--- src/CMakeLists.txt.orig	2023-01-18 15:57:53.000000000 -0600
+++ src/CMakeLists.txt	2023-01-23 04:08:02.000000000 -0600
@@ -128,7 +128,6 @@
 
 if (${CMAKE_CXX_COMPILER_ID} STREQUAL GNU)
     # Do not check format overflows on this file, to skip a false positive warning
-    set_source_files_properties(value.cpp PROPERTIES COMPILE_FLAGS -Wno-format-overflow)
 endif()
 
 set_target_properties( exiv2lib PROPERTIES
